fastlane.gemspec

```diff
@@ -26,9 +26,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'slack-notifier', '~> 1.0' # Slack notifications
   spec.add_dependency 'shenzhen', '>= 0.12.1' # to upload to Hockey and Crashlytics
```

Comment:
Yes, but I wasn't sure if I should do it, or wait for @KrauseFx to fix it and then merge in the fix.